source/enumerate-collections.rst

```diff
@@ -302,6 +302,22 @@ result for ``system.namespaces``.
 The returned result for each variant MUST be equivalent, and each collection
 that is returned MUST use the field names ``name`` and ``options``.
 
+In MongoDB 4.4, the ``ns`` field was removed from the index specifications,
+including in the ``idIndex`` option returned from the ``listCollections``
```

Comment:
`idIndex` isn't a collection option / it isn't part of the returned `options` subdocument, so this should probably be reworded. Maybe replace "including" onwards with something like: "so the index specification included in the `idIndex` field of the collection information will no longer contain an `ns` field". 

This distinction made me realize that this spec is a little out of date, since it seems to indicate that the only things returned from `listCollections` are "name/option" pairs, which of course isn't true given that something like `idIndex` exists. This is just a holdover from when listing collections involved querying the `<db>.system.namespaces` collection. I filed [SPEC-1596](https://jira.mongodb.org/browse/SPEC-1596) to discuss these issues and work towards updating this spec. 

One thing to note is that until that ticket is addressed, the following line _technically_ means that `idIndex` shouldn't be returned from this method at all (though I have a strong feeling most drivers _do_ in fact report these extra fields):
>The returned result for each variant MUST be equivalent, and each collection that is returned MUST use the field names name and options.

Reply:
Can we get this resolved and the PR approved?

Reply:
I've reworded the sentence as suggested. I can work on rewording the other sections to bring the spec up-to-date in a separate pull request pending outcome of discussion in SPEC-1596.